Services/Common/OJS.Services.Common/Implementations/ContestResultsAggregatorCommonService.cs

```diff
@@ -56,13 +57,17 @@ public ContestResultsViewModel GetContestResults(ContestResultsModel contestResu
                                              contestResultsModel.Official)
                                          .Count();
 
+        contestResults.PagesCount = totalParticipantsCount > contestResultsModel.ItemsPerPage
+            ? (int)Math.Ceiling((double)totalParticipantsCount / contestResultsModel.ItemsPerPage)
+            : 1;
```

Comment:
Can we use `ToPagedResultAsync` from `QueryableExtensions`?

Reply:
Done.